marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/entities/entity-search-lib.sjs

```diff
@@ -535,11 +540,58 @@ function isHubEntityInstance(docUri) {
   return isHubEntityInstance;
 }
 
+function getRecordHistory(docUri) {
+  const history = [];
+  const provResults = xdmp.invokeFunction(function () {
```

Comment:
Only this part needs to be amped. And we have a natural home for it - in prov.sjs. But let's not put it in the Provenance class (I don't think we can amp it either if we do that). Just make a new exported function in there. 

To keep it flexible, make "out" (if that is really "output", let's call it "output" - abbreviations rarely buy us anything) an argument to the function. So how about - findProvenance(uri, relations). 

Then, make the above amp "findProvenance.json" - no need for "amp" or "dhf" in the filename.

I think it's worth putting the invokeFunction in the amped function too, so that the client doesn't have to worry about doing that part.